Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/AtomicSequences/Validation/EconDamperPositionLimitsMultiZone_LoopDisable.mo

```diff
@@ -0,0 +1,183 @@
+within Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.AtomicSequences.Validation;
+model EconDamperPositionLimitsMultiZone_LoopDisable
+  "Validation model for the multiple zone VAV AHU minimum outdoor air control - damper position limits"
+  extends Modelica.Icons.Example;
+
+  parameter Types.FreezeProtectionStage freProStage1 = Types.FreezeProtectionStage.stage1
+    "Freeze protection stage 1";
+  parameter Types.FreezeProtectionStage freProStage2 = Types.FreezeProtectionStage.stage2
+    "Freeze protection stage 2";
+  parameter Integer freProStage1Num = Integer(freProStage1)-1
```

Comment:
I see. I committed to this branch a new package
`Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.Constants.FreezeProtectionStages`. Please use this and delete the enumeration. The new package assigns the integer value `0` to `stage0`, etc.